src/getstarted/TF_Dashboard/TF_Dashboard.md

```diff
@@ -1,16 +1,3 @@
-<h1> Threefold Dashboard: Create an Account and Transfer TFT</h1>
```

Comment:
Why do we remove the Title, the table of contents and the embedded video tutorial?

I do not understand the need to "change" the section by removing the work that was done.

Reply:
![image](https://user-images.githubusercontent.com/62248851/228238048-bbd87f07-8d6b-4fb2-bc33-fd3d0285929d.png)

this is my suggestion, and we will add the video before the table of contents